Tools/AP_Bootloader/asm_arm.inc

```diff
@@ -0,0 +1,2455 @@
+#define DEC_5 4
```

Comment:
Where did this come from?

Reply:
This is from uECC. https://github.com/kmackay/micro-ecc